### PR TITLE
2024-03-04 TSC Meeting minutes

### PR DESCRIPTION
**Pull request by weierophinney**
- Adds minutes from 2024-03-04 meeting
- Resets agenda for April meeting

### CODE REVIEW

File: meetings/agenda.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
 # Next Technical Steering Committee Meeting Agenda
 
-- Date: 2024-01-08
+- Date: 2024-04-01
```
Comment (michalbundyra) [inline meetings/agenda.md:3]: as noted at the very end - it's gonna be Easter Monday and holidays in Europe (and Australia), maybe better postpone

**Comment by ALTAMASH80** [comment]
Hi, is there any chance for someone to join and listen to what will be discussed in the meeting or it is for members only? Thanks!

**Comment by akrabat** [comment]
> Hi, is there any chance for someone to join and listen to what will be discussed in the meeting or it is for members only? Thanks!

The TSC meetings are held in text message form on Slack in the `#tsc-meeting` channel on the 1st Monday of each month at 19:00 UTC (unless moved a week for bank holiday or absence type reasons). You're very welcome to join and participate in the conversations.

**Comment by ALTAMASH80** [comment]
Thanks! I'll try to find this channel and join it. I hope free slack users can join.

File: meetings/minutes/2024-03-04-TSC-Minutes.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  - George Steel
+  - Luís Cobucci
+  - Marco Pivetta
+  - Matthew Setter
+  - Matthew Weier O'Phinney
+  - Maximilian Bösing
+  - Michał Bundyra
+  - Rob Allen
+
+Quroum WAS met (12 out of 15 members were present).
```
Comment (froschdesign) [inline meetings/minutes/2024-03-04-TSC-Minutes.md:20]: Here are 13 members listed and in the [GitHub team named "Technical Steering Committee"](https://github.com/orgs/laminas/teams/technical-steering-committee) there are 17 members + the Laminas Bot?! 🤔

Comment (Xerkus) [reply meetings/minutes/2024-03-04-TSC-Minutes.md:20]: 17 members https://github.com/laminas/technical-steering-committee/blob/main/MEMBERS.md

Comment (weierophinney) [reply meetings/minutes/2024-03-04-TSC-Minutes.md:20]: There's a few folks on that list who have indicated they are no longer active; we just haven't done the cleanup at this point.

I'm updating it to read 13 for attendees, however.

**Pull request merged**